adoc/admin-updates.adoc

```diff
@@ -19,15 +19,19 @@ Updating of {kube} components is handled via `skuba`.
 
 The general procedure should look like this:
 
-. Update the packages and reboot your management workstation to get all the latest changes to skuba and it's dependencies
-. Run `skuba addon upgrade apply` on the management workstation
-.
-. Check if there are addon upgrades available for the current cluster
-. Upgrade all master nodes
-. Upgrade all worker nodes
-. Check if new addons are available for the new version
-.. If necessary upgrade master nodes again
-.. If necessary upgrade worker nodes again
+. Update the packages and reboot your management workstation to get all the latest changes to skuba, helm and their dependencies
```

Comment:
Before this - as a first step - check if all applications and  deployments running in the cluster are supported in their current version with the next release of k8s the cluster is going to be upgraded to.
In case the current deployed application is i.e. not compatible with the next k8s release - update the application before upgrading the cluster to ensure application availability.